test_strenum.py

```diff
@@ -1,10 +1,13 @@
 from enum import auto
+
+import pytest
+
 from strenum import StrEnum
 
 
 class HttpMethod(StrEnum):
     GET = auto()
-    HEAD = auto
+    HEAD = auto()
```

Comment:
I think this was a bug? Or at least should have been.

Reply:
Same typo in README.md https://github.com/irgeek/StrEnum/pull/2

Reply:
Thanks, fixed.